core/types/deposit_tx.go

```diff
@@ -79,9 +81,9 @@ func (tx *DepositTx) nonce() uint64          { return DepositsNonce }
 func (tx *DepositTx) to() *common.Address    { return tx.To }
 
 func (tx *DepositTx) rawSignatureValues() (v, r, s *big.Int) {
-	panic("deposit tx does not have a signature")
+	return zeroBig, zeroBig, zeroBig
```

Comment:
Can you point to the code that calls this function? There is no signature, so it should not be called in the first place

Reply:
https://gist.github.com/tuxcanfly/189f285c82d12c059aef20142b225371